lsp/src/text_document.ml

```diff
@@ -159,3 +164,59 @@ let apply_content_changes ?version t changes =
     | Some version -> { document with version }
   in
   { t with document }
+
+type change =
+  { start : int
+  ; stop : int
```

Comment:
mention `start` and `stop` are offsets would be useful

Reply:
I'll send another PR to refactor this stuff.